scm/driver/gitea/user.go

```diff
@@ -37,7 +37,7 @@ func (s *userService) FindEmail(ctx context.Context) (string, *scm.Response, err
 // native data structures
 //
 
-type user struct {
+type User struct {
```

Comment:
when changing from `user` to `User` we are changing from private to public.  Since most data structures are internal to the package they should be private.  Any reason to make these public?

Reply:
That was my bad, reverted.